.github/pull_request_template.md

```diff
@@ -0,0 +1,49 @@
+## Description
+
+<!--
+Describes the nature of your changes. If they are substantial, you should
+further subdivide this into a section describing the problem you are solving and
+another describing your solution.
+-->
+
+## Related issue(s)
+
+<!--
+- Short description of how the PR relates to the issue, including an issue link.
+For example
+- Fixed #100500 by adding lenses to exported items
+
+Write 'None' if there are no related issues (which is discouraged).
+-->
+https://github.com/serokell/tezos-client/issues/
+
+<!--
+Ideally a PR has all of the checkmarks set.
+
+If something in this list is irrelevant to your PR, you should still set this
+checkmark indicating that you are sure it is dealt with (be that by irrelevance).
+
+If you don't set a checkmark (e. g. don't add a test for new functionality),
+you must be able to justify that.
+-->
+
+#### Related changes (conditional)
+
+- Tests
+  - [ ] If I added new functionality, I added tests covering it.
+  - [ ] If I fixed a bug, I added a regression test to prevent the bug from
+        silently reappearing again.
+
+- Documentation
+  - [ ] I checked whether I should update the docs and did so if necessary:
+    - [README](../tree/master/README.md)
+    - Haddock
+
+- Public contracts
+  - [ ] Any modifications of public contracts comply with the [Evolution
+  of Public Contracts](https://www.notion.so/serokell/Evolution-of-Public-Contracts-2a3bf7971abe4806a24f63c84e7076c5) policy.
+  - [ ] I added an entry to the [changelog](../tree/master/CHANGES.md) if my changes are visible to the users
```

Comment:
1. Well, `CHANGES.md` also doesn't exist. let's add dummy `CHANGES.md` or remove this line.
2. Please review this list carefully. E. g. why do we mention Haddock here? Are we going to have tests? How does one write tests for nix? Why do we care about evolution of public contracts? As I said, it is not a library, it doesn't have any API. I believe this checklist can be greatly simplified.

Reply:
Actually, it seems that only `README.md` makes sense at the current point